raft/README.md

```diff
@@ -71,6 +71,19 @@ To start a node from scratch:
   n := raft.StartNode(c, []raft.Peer{{ID: 0x02}, {ID: 0x03}})
 ```
 
+To allow a single node cluster, set peers to itself, like so:
+
+```go
+	peers := []raft.Peer{{ID: 0x01}}
+	n := raft.StartNode(c, peers)
+```
+
+And to allow a new node to join this cluster, don't pass in any peers. Then `ProposeConfChange` in the existing cluster to add this new node.
```

Comment:
And to allow a new node to join an existing cluster, do not pass in any peers. First, you need add the node to the existing cluster by calling `ProposeConfChange` on any existing node inside the cluster. Then, you can start the node like with empty peer list like: